geode-core/src/main/java/org/apache/geode/cache/control/ResourceManager.java

```diff
@@ -77,6 +78,21 @@
    */
   Set<RebalanceOperation> getRebalanceOperations();
 
+  /**
+   * Creates a class for defining and starting restore redundancy operations.
```

Comment:
I think we should improve these javadocs and maybe clarify the difference between "restore redundancy" and rebalance. Restore redundancy is really just a subset of the rebalance operation - there is no need to invoke both, for example.

Reply:
How would this be: